test/postgresql.autoupdate.test.js

```diff
@@ -677,4 +679,274 @@ describe('autoupdate', function() {
       });
     });
   });
+
+  describe('multicolumn indexes on table in schema', () => {
+    const schema = {
+      name: 'Person',
+      options: {
+        postgresql: {
+          schema: 'public',
+          table: 'Person',
+        },
+        indexes: {
+          uniqueNameIndex: {
+            keys: {'firstName': 1, 'lastName': 1},
+            options: {unique: false},
+          },
+        },
+      },
+      properties: {
+        firstName: {
+          type: 'string',
+        },
+        lastName: {
+          type: 'string',
+        },
+        middleName: {
+          type: 'string',
+        },
+      },
+    };
+    const changedSchema = {
+      ...schema,
+      options: {
+        ...schema.options,
+        indexes: {
+          uniqueNameIndex: {
+            keys: {'firstName': -1, 'lastName': -1},
+            options: {unique: true},
+          },
+        },
+      },
+    };
+    const DROP_INDEX_REGEX = /DROP INDEX.*uniqueNameIndex/;
+
+    let personModel;
+
+    afterEach(async () => {
+      await personModel.destroyAll();
+    });
+
+    describe('when table has no existing indexes', () => {
+      it('updates without errors', async () => {
+        personModel = ds.define(schema.name, schema.properties, schema.options);
+        await ds.autoupdate([schema.name]);
+        assert(true, true);
+      });
+    });
```

Comment:
I am a bit confused here. Does this test mean that when we first create the table and it doesn't have existing indexes in the DB?

Reply:
@agnes512 Essentially this is just performing the same test as the single column index test here: https://github.com/strongloop/loopback-connector-postgresql/blob/09791c6ec7b502d285f6a70202eee91dd92e9092/test/postgresql.autoupdate.test.js#L662
I could add assertions to both tests to further validate that the indexes exist on the models following the `autoupdate()` calls (or at least add it to my test.) The intent of the test as-is, though, is just to check that no errors are thrown.